FirebasePerformance/Sources/FPRProtoUtils.h

```diff
@@ -1,65 +1,65 @@
-// Copyright 2020 Google LLC
+//// Copyright 2020 Google LLC
```

Comment:
2021 - copyrights should have the year they're first published externally

Reply:
FPRProtoUtils.h was first published externally in 2020. I will remove this file and all proto related files in the next PR.